internal/twirptest/spec_compatibility/compatibility_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Twitch Interactive, Inc.  All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not
+// use this file except in compliance with the License. A copy of the License is
+// located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package spec_compatibility
```

Comment:
That was where it was! Check the comments above, haha!

@dpolansky asked me to make it into another package.

Reply:
Oh, sorry for the confusion. Yes I read that comment but I think it was based on previous context (the PR had a larger scope before). If it is still not a good idea (or I didn't understand the comment well), then just rename this test. No big deal :P

Reply:
The general vibe I got was all the future tests that will be necessary to ensure spec compatibility will go into that package as well.

Reply:
I've moved it back.

Reply:
> The general vibe I got was all the future tests that will be necessary to ensure spec compatibility will go into that package as well.

I see. But I would argue that is not a good way to group tests (this may be just my opinion). You may end up having only one specific test (non-camel-cased-names) under a generic umbrella term (spec-compat). Or you may end up having too many unrelated tests under the same umbrella term. We just don't know, so it is usually best not to try guessing the future too much, and just name things for what they are. If things change too much in the future, well, then we should refactor, but it is better to refactor later, at the right time, when you know what works better. A good rule of thumb to know when it's time to refactor is the [rule of three](https://blog.codinghorror.com/rule-of-three/) 🤓